### PR TITLE
fix null value problem

### PR DESCRIPTION
**Pull request by EightMinuteYouth**

### CODE REVIEW

File: lib/src/router_delegate.dart [hunk 1]
```diff
@@ -192,10 +192,14 @@ class FFRouterDelegate extends RouterDelegate<RouteSettings>
     required String name,
     Map<String, dynamic>? arguments,
   }) {
-    FFPage<T?> ffPage =
-        getRouteSettings(name: name, arguments: arguments).toFFPage<T>(
+    final FFRouteSettings settings = getRouteSettings(name: name, arguments: arguments);
```
Comment (AlexV525) [inline lib/src/router_delegate.dart:195]: Use 80 characters line wrap.

```suggestion
    final FFRouteSettings settings = getRouteSettings(
      name: name,
      arguments: arguments,
    );
```

File: lib/src/router_delegate.dart [hunk 2]
```diff
@@ -192,10 +192,14 @@ class FFRouterDelegate extends RouterDelegate<RouteSettings>
...
+
+    FFPage<T?> ffPage = settings.toFFPage<T>(
       key: getUniqueKey(),
+      /// If the current is 404, default SizedBox
```
Comment (AlexV525) [inline lib/src/router_delegate.dart:199]:
```suggestion
      // If the current route results nothing, fallback to a `SizedBox`.
```

File: lib/src/router_delegate.dart [hunk 3]
```diff
@@ -192,10 +192,14 @@ class FFRouterDelegate extends RouterDelegate<RouteSettings>
...
+      widget: settings.widget ?? const SizedBox()
```
Comment (AlexV525) [inline lib/src/router_delegate.dart:200]: Why do we need `SizedBox` since `widget` is nullable?

**Comment by zmtzawqlp** [comment]
https://github.com/fluttercandies/ff_annotation_route_library/commit/1ef6df7e4a3656dd09a07118c0730a79def5c7d3

add notfindWidget, you can set it if page is not found

**Pull request closed without merging**